src/Servers/IIS/AspNetCoreModuleV2/CommonLib/HostFxrResolver.cpp

```diff
@@ -93,7 +94,12 @@ HostFxrResolver::GetHostFxrParameters(
             LOG_INFOF(L"Checking application.dll at '%ls'", applicationDllPath.c_str());
             if (!is_regular_file(applicationDllPath))
             {
-                throw InvalidOperationException(format(L"Application .dll was not found at %s", applicationDllPath.c_str()));
+                errorContext.subStatusCode = 38;
+                errorContext.errorReason = "Application Dll not found. Confirm the application dll is present and is not published as a single executable.";
```

Comment:
Suggesting saying `DLL` because it's an abbreviation. You'd only use `Dll` or `dll` when it's code (e.g. a variable name).

Reply:
@Eilon is superior word-smith